docs/migration-guide.en.md

```
@@ -1,36 +1,34 @@
 
-```bash
-# Path A: Docker pull from ghcr.io (simplest)
-docker pull ghcr.io/vencil/da-tools:v2.8.0
-
-# Path B: Download a static binary to PATH
-curl -fsSLo da-guard.tar.gz https://github.com/vencil/Dynamic-Alerting-Integrations/releases/download/tools/v2.8.0/da-guard-linux-amd64.tar.gz
-tar xzf da-guard.tar.gz && sudo install -m 0755 da-guard-linux-amd64 /usr/local/bin/da-guard
-```
+| You are… | Start with | Drill into |
+|---|---|---|
+| Tenant tech lead | [Where Are You?](#where-are-you-你在哪個階段), [§3 Generate config](#3-generate-tenant-config-da-tools-scaffold), [§9 Dimensional labels](#9-dimensional-labels-multi-db-type-support) | [§7 Verification](#7-post-migration-verification), [§12 FAQ](#12-faq) |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)